nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -213,28 +216,42 @@ def _statistic_collector_builder(collector_name: str,
     @staticmethod
     def _create_quantizer(quantizer_config: QuantizerConfig,
                           scale_shape: Tuple,
-                          statistics: MinMaxTensorStatistic) -> BaseQuantizer:
-        quantizer_cls = QUANTIZATION_MODULES.get(quantizer_config.mode)
+                          parameters: FakeQuantizeParameters,
+                          target_type: TargetType) -> BaseQuantizer:
+        mode = quantizer_config.mode
+        quantizer_cls = QUANTIZATION_MODULES.get(mode)
+        narrow_range = target_type == TargetType.OPERATION_WITH_WEIGHTS and mode == QuantizationMode.SYMMETRIC
         quantizer_spec = PTQuantizerSpec.from_config(quantizer_config,
-                                                     narrow_range=False,
+                                                     narrow_range=narrow_range,
                                                      scale_shape=scale_shape,
                                                      half_range=False,
                                                      logarithm_scale=False,
                                                      is_quantized_on_export=False,
                                                      compression_lr_multiplier=None)
         quantizer = quantizer_cls(quantizer_spec)
+
         # Fill it with minmax
-        quantizer.apply_minmax_init(min_values=statistics.min_values,
-                                    max_values=statistics.max_values)
+        PTMinMaxAlgoBackend._fill_quantizer_parameters(quantizer, parameters)
         return quantizer
 
+    @staticmethod
+    def _fill_quantizer_parameters(quantizer: BaseQuantizer, parameters: FakeQuantizeParameters) -> None:
+        quantizer.eps = 0
+        if isinstance(quantizer, AsymmetricQuantizer):
+            quantizer.input_low = torch.nn.Parameter(torch.from_numpy(parameters.input_low))
+            quantizer.input_range = \
+                torch.nn.Parameter(torch.from_numpy(np.array(parameters.input_high - parameters.input_low)))
+        else:
+            quantizer.signed = np.any(parameters.input_low < 0)
+            quantizer.scale = torch.nn.Parameter(torch.from_numpy(parameters.input_high))
```

Comment:
It looks like the formula should be 

```
quantizer.scale = torch.nn.Parameter(torch.from_numpy(np.maximum(np.abs(parameters.input_high), np.abs(parameters.input_low)))
```
@l-bat Could you double check?

Reply:
Before the change, did you check that the general FQ parameters match the FQ parameters of the ONNX model after exporting the torch model? Maybe you have a test?